stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -825,18 +825,39 @@ public class ChatClient internal constructor(
     }
 
     @CheckResult
-    public fun getReplies(messageId: String, limit: Int): Call<List<Message>> {
-        return api.getReplies(messageId, limit)
-    }
+    @InternalStreamChatApi
+    @ExperimentalStreamChatApi
+    public fun getRepliesInternal(messageId: String, limit: Int): Call<List<Message>> = api.getReplies(messageId, limit)
```

Comment:
Copy methods without side effects for bakcward compatibility when use `ThreadController`. Will be remove when `ChatDomain` and its' controllers are deprecated and hidden from public API